tests/protocol/stun/stun.zeek

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+# @TEST-EXEC: set >envs
+# @TEST-EXEC: ${ZEEK} -NN >zeek 2>&1
+# @TEST-EXEC: ${ZEEK} -Cr ${TRACES}/stun-ice-testcall.pcap %INPUT
```

Comment:
We need to document the provenience of this pcap file.

Reply:
It's from the original repo, that I already documented.

Reply:
Could you add an entry to `tests/Traces/README`?

Reply:
The STUN readme isn't enough?

Reply:
Added